context.go

```diff
@@ -186,6 +186,16 @@ func (c *Context) AbortWithError(code int, err error) *Error {
 	return c.Error(err)
 }
 
+// GraceError calls `Error` internally.
+// This method does not panic when err is nil, and no return value.
+// See Context.Error() for more details.
+func (c *Context) GraceError(err error) {
+	if err == nil {
+		return
+	}
+	c.Error(err) // nolint: errcheck
+}
+
```

Comment:
These changes just to ignore error check like golint tool?

Reply:
yes

Reply:
I think this is useless because when we get a error from a function,we need judge whether the error is `nil` 。

Reply:
@bestgopher Looks like the function c.Error(error) never returns nil => it is not for errcheck